shared/log.rs

```diff
@@ -32,6 +34,30 @@ pub fn get_log_file_handles(task_id: usize, settings: &Settings) -> Result<(File
     Ok((stdout, stderr))
 }
 
+fn command_data_to_text(mut data: &[u8]) -> Cow<str> {
```

Comment:
I'm not 100% sure about the name `command_data_to_text`, `command_data` seems a little vague.
Maybe something like `process_output_to_text`?